src/Components/Spotify/SpotifyAuth.js

```diff
@@ -0,0 +1,13 @@
+import spotifyConfig from "../../Config/spotify";
+
+
+const SpotifyAuth = function () {
+  return (
+    <a className={""}
+       href={`${spotifyConfig.authEndpoint}?client_id=${spotifyConfig.clientId}&redirect_uri=${spotifyConfig.redirectUri}&scope=${spotifyConfig.scopes.join("%20")}&response_type=token&show_dialog=true`}>
```

Comment:
Could be made a bit more readable using URLSearchParams:

```javascript
  const urlParams = new URLSearchParams({
    client_id: spotifyConfig.clientId,
    redirect_uri: spotifyConfig.redirectUri,
    scope: spotifyConfig.scopes.join(" "),
    response_type: "token",
    show_dialog: "true"
  });
```

and then

```
href={`${spotifyConfig.authEndpoint}?${urlParams.toString()}`}
```

Reply:
Good idea!